amgut/lib/data_access/ag_data_access.py

```diff
@@ -386,7 +386,8 @@ def deleteSample(self, barcode, ag_login_id):
             sql = """UPDATE ag_kit_barcodes
                      SET participant_name = NULL, site_sampled = NULL,
                          sample_time = NULL, sample_date = NULL,
-                         environment_sampled = NULL, notes = ''
+                         environment_sampled = NULL, notes = NULL,
```

Comment:
:+1:

On Mon, Feb 29, 2016 at 3:55 PM, Joshua Shorenstein <
notifications@github.com> wrote:

> In amgut/lib/data_access/ag_data_access.py
> <https://github.com/biocore/american-gut-web/pull/573#discussion_r54501230>
> :
>
> > @@ -386,7 +386,8 @@ def deleteSample(self, barcode, ag_login_id):
> >              sql = """UPDATE ag_kit_barcodes
> >                       SET participant_name = NULL, site_sampled = NULL,
> >                           sample_time = NULL, sample_date = NULL,
> > -                         environment_sampled = NULL, notes = ''
> > +                         environment_sampled = NULL, notes = NULL,
>
> No, notes can be null, so this is fixing it so that it is consistent with
> a non-logged barcode instead of slightly different with the empty string.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/american-gut-web/pull/573/files#r54501230>.
>